tools/bazel/postinstall-patches.js

```diff
@@ -118,9 +118,7 @@ searchAndReplace(`[formatProperty + "_ivy_ngcc"]`, '[formatProperty]',
 
 // Workaround for https://github.com/angular/angular/issues/33452:
 searchAndReplace(/angular_compiler_options = {/, `$&
-        "strictTemplates": True,
-        "strictDomLocalRefTypes ": False,
-        "strictDomEventTypes": False,`, 'node_modules/@angular/bazel/src/ng_module.bzl');
+        "strictTemplates": True,`, 'node_modules/@angular/bazel/src/ng_module.bzl');
```

Comment:
Can you increment the patch number? That will ensure that this edit is not accidentally skipped when someone simply runs `yarn` on existing `node_modules`.

Reply:
Done.